# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/external-memory/src/fd.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/external-memory/src/fd.rs b/third_party/rust/external-memory/src/fd.rs
deleted file mode 100644
--- a/third_party/rust/external-memory/src/fd.rs
+++ /dev/null
@@ -1,20 +0,0 @@
-#[derive(Debug)]
-/// Unix file descriptor.
-pub struct Fd(i32);
-impl From<i32> for Fd {
-    fn from(fd: i32) -> Self {
-        Self(fd)
-    }
-}
-#[cfg(unix)]
-impl std::os::unix::io::AsRawFd for Fd {
-    fn as_raw_fd(&self) -> std::os::unix::io::RawFd {
-        self.0
-    }
-}
-impl std::ops::Deref for Fd {
-    type Target = i32;
-    fn deref(&self) -> &Self::Target {
-        &self.0
-    }
-}